src/renderer/api/kube-watch-api.ts

```diff
@@ -64,36 +64,51 @@ export class KubeWatchApi {
     let isUnsubscribed = false;
 
     const load = (namespaces = subscribingNamespaces) => this.preloadStores(stores, { namespaces, loadOnce });
-    let preloading = preload && load();
+    let preloading: boolean | ReturnType<typeof load> = preload && load();
```

Comment:
This is the correct typing of this value. I assume that because we aren't using `strict: true` it was typed as only `ReturnType<typeof load>`